redash/authentication/__init__.py

```diff
@@ -239,6 +239,7 @@ def setup_authentication(app):
     login_manager.anonymous_user = models.AnonymousUser
 
     app.secret_key = settings.COOKIE_SECRET
+    app.config.update(REMEMBER_COOKIE_SECURE=True, REMEMBER_COOKIE_HTTPONLY=True)
```

Comment:
Does `REMEMBER_COOKIE_SECURE` works with `http`?

Reply:
Couldn't find a better way to test for HTTPS at this level than [this](https://github.com/getredash/redash/pull/3214/files#diff-f42630f8986c45dd8f9a209502bf440fR242). The caveat of this is that if someone sets `ENFORCE_HTTPS` to false, but still hosts their Redash over HTTPS, they won't get secure cookies. WDYT?

Reply:
What do you think about the following suggestion:

1. Always set `REMEMBER_COOKIE_HTTPONLY` to True (I'm assuming this works both on HTTP and HTTPS, right?).
2. Have an environment variable to control `REMEMBER_COOKIE_SECURE` that has the default value of whatever `ENFORCE_HTTPS` is set to, but whoever wants can override it.

?

Reply:
Ping.

Reply:
Yeah, I guess that makes sense. Can't think of why someone would host over HTTPS and avoid `ENFORCE_HTTPS=true` though 🤷‍♂